tests/clv/test_distributions.py

```diff
@@ -154,3 +154,102 @@ def test_continuous_contractual_sample_prior(
             prior = pm.sample_prior_predictive(samples=100)
 
         assert prior["prior"]["cc"][0].shape == (100,) + expected_size
+
+
+class TestParetoNBD:
+    @pytest.mark.parametrize(
+        "value, r, alpha, s, beta, T, logp",
+        [
+            (np.array([5, 6.3]), 0.55, 10.58, 0.61, 11.67, 12, -13.851591),
+            (
+                np.array([5, 6.3]),
+                [0.45,.55],
+                10.58,
+                0.61,
+                11.67,
+                12,
+                [-14.180599, -13.851591],
```

Comment:
Where did you get the expected logps? I would mention that as a comment (e.g. obtained from lifetimes, some R package, whatever) so that future maintainers can check/ obtain new values if parameters change or something breaks.

Reply:
The likelihood function variant here is not being used by other libraries, so the `logp` values have no basis for comparison. What's important is that it recovers the same parameter values, which I'll be testing for when I start work on the Pareto/NBD model.